web_app/api/position.py

```diff
@@ -74,9 +75,10 @@ async def close_position(position_id: str) -> str:
     Close a position.
     :param position_id: contract address
     :return: str
+    :raises: HTTPException :return: Dict containing status code and detail
     """
-    if position_id is None or position_id == 'undefined':
-        raise ValueError("Invalid position_id provided")
+    if position_id is None or position_id == "undefined":
+        raise HTTPException(status_code=404, detail="Invalid position_id provided")
```

Comment:
I would rather change it to `detail=Position not Found` it makes more sense to me